### DISCUSSION TITLE
Problem with building the docker

### DISCUSSION THREAD

**Discussion by Inshu32** [stella-cv/stella_vslam#601] [Q&A] [answered]
I have tried building the docker file using the given commands across four different linux installations. Pop_os, Ubuntu 24, Fedora 40and Debian12 doc(all latest versions). I am getting the same error on step 18/20.  docker versions is 26,1,3 build b27abbb
The error code is 

ERROR: failed to solve: process "/bin/sh -c set -x &&   git clone https://github.com/stella-cv/stella_vslam_examples.git &&   cd stella_vslam_examples &&   git checkout -q ${EXAMPLES_COMMIT} &&   git submodule update --init --recursive &&   mkdir -p build &&   cd build &&   cmake     -DCMAKE_BUILD_TYPE=RelWithDebInfo     -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}     -DUSE_STACK_TRACE_LOGGER=ON     .. &&   make -j${NUM_THREADS}" did not complete successfully: exit code: 2

**Comment by ymd-stella** [comment]
It did not reproduce. Mention is not required.

**Reply by Inshu32** [reply]
What should I do. I need it working asap because I need to use it in one my projects. tried other SLAM's too but having dependencies issue.

**Reply by Inshu32** [reply]
Hi any updates regarding this ?

**Reply by ymd-stella** [reply]
There is nothing I can do as I cannot reproduce it. You should identify what is failing in the command.

**Reply by Inshu32** [reply]
Is it working on your system then ? Can you tell the config where you could install the docker, I will also try using the same linux or windows. The docker is failing in step 18/20 while resolving examples. Can you tell me how to identify ?

**Reply by ymd-stella** [reply, marked as answer]
See https://github.com/stella-cv/stella_vslam/issues/605

**Reply by Inshu32** [reply]
Thanks, it worked on freshly installed Unbuntu.
